vacasa/connect/connect.py

```diff
@@ -732,12 +737,14 @@ def create_reservation(self,
                 booked in ISO 4217 alpha code format (e.g. 'USD', 'CLP', etc.).
             display_currency_code: The currency preference of the guest in
                 ISO 4217 alpha code format (e.g. 'USD', 'CLP', etc.).
+            headers: custom headers to be passed in and added to the dictionary
+                of common headers created by the _headers() method
 
         Returns:
             A response object as a dict
         """
         url = f"{self.endpoint}/v1/reservations"
-        headers = self._headers()
+        headers.update(self._headers())
```

Comment:
This will throw an `AttributeError` if `headers = None` which is the default value.

Reply:
This should be fixed now.